wg-meetings/2016-06-01.md

```diff
@@ -0,0 +1,91 @@
+# Node.js Tracing/Diag WG Meeting
+
+- Date: 2016-06-01
+- YouTube: http://www.youtube.com/watch?v=tNxYDvQ1kT8
+
+## Attendees
+
+- @joshgav
+- @mhdawson
+- @matthewloring
+- @ofrobots
+- @paulirish
+- @pavelfeldman
+- @caseq
+- @jasonlaster
+- @pmuellr
+- @fmeawad
++ several more via YouTube
+   
+## Agenda
+
+- TRACE_EVENT - Google's plans and how Node can build on them (@ofrobots)
+- Trace module - @joshgav's proposal - why and what.
+    - [nodejs/tracing-wg#50], [joshgav/node-trace]
+- Chrome Debugging Protocol and v8_inspector (@ofrobots, Google)
+    - [nodejs/node#6792], [nodejs/node#7072]
+    - What it brings to Node and the community now and in the future.
+    - How can the Diagnostics WG utilize it?
+- AsyncWrap [deferred]
+- Expanding scope of this WG [deferred]
+
+## Next Meeting
+
+- Return to monthly cadence - first Wednesday of each month at 12pm US Pacific (immediately before CTC meeting). Next meeting will be July 6.
+
+## Minutes
+
+### Tracing
+
+In Chromium, the Trace Event trace system is implemented through a Tracing Controller. @ofrobots and @matthewloring are working on porting this implementation to V8/D8.
+
+In this Tracing Controller, traces are not streamed, they are collected in memory and dumped on request. It would be up to Node to determine how to flush, stream, etc. in Node.
+
+chrome://tracing is intended for platform developers while “Timeline” view in Chrome DevTools is intended for web/app developers. Data source is the same but filtered differently for each tool.
+
+Instead of “structural tracing” through explicit trace points in code, Google prefers sampling all code. Although slightly less exact on timings this provides more thorough perspective of operations. However, Google hasn’t yet integrated this sampling-based tracing into the default trace configuration.
```

Comment:
Maybe we should call out "for JavaScript code" because IIRC for all native code explicit trace points are used.